lib/typescript/httpclient/BUILD

```diff
@@ -0,0 +1,16 @@
+load("@com_github_airyhq_bazel_tools//web:typescript.bzl", "ts_library")
+
+package(default_visibility = ["//visibility:public"])
+
+ts_library(
+    name = "httpclient",
+    deps = [
+        "//lib/typescript/types",
+        "@npm//@types/node",
+        "@npm//lodash-es",
+        "@npm//react",
+        "@npm//redux",
+        "@npm//redux-starter-kit",
+        "@npm//typesafe-actions",
+    ],
```

Comment:
unless I'm misunderstanding the code, we shouldn't need more than the types as deps here?

Reply:
yes totally, forgot to delete those